README.tr.md

```
@@ -0,0 +1,485 @@
+
+1. **Verinizin kontrolü sizde.** CV'niz, iletişim bilgileriniz ve kişisel verileriniz kendi makinenizde kalır ve doğrudan seçtiğiniz yapay zekâ sağlayıcısına (Anthropic, OpenAI vb.) gönderilir. Verilerinizi toplamıyor, saklamıyor veya bunlara erişimimiz yok.
+2. **Yapay zekânın kontrolü sizde.** Varsayılan istemler yapay zekâya başvuruları otomatik göndermemesini söyler, ancak yapay zekâ modelleri öngörülemez davranabilir. İstemleri değiştirir veya farklı modeller kullanırsanız, bunu kendi sorumluluğunuzda yaparsınız. **Göndermeden önce yapay zekâ tarafından üretilen içeriğin doğruluğunu her zaman kontrol edin.**
+3. **Üçüncü taraf hizmet koşullarına uyarsınız.** Bu aracı, etkileşimde bulunduğunuz kariyer portallarının (Greenhouse, Lever, Workday, LinkedIn vb.) Kullanım Koşullarına uygun şekilde kullanmalısınız. Bu aracı işverenlere spam göndermek veya ATS sistemlerini aşırı yüklemek için kullanmayın.
+4. **Garanti yoktur.** Değerlendirmeler öneridir, gerçek değildir. Yapay zekâ modelleri beceri veya deneyim hakkında yanılsama üretebilir. Yazarlar; istihdam sonuçlarından, reddedilen başvurulardan, hesap kısıtlamalarından veya başka herhangi bir sonuçtan sorumlu değildir.
+
+Tüm ayrıntılar için bkz. [LEGAL_DISCLAIMER.md](LEGAL_DISCLAIMER.md). Bu yazılım, herhangi bir garanti olmaksızın "olduğu gibi" [MIT Lisansı](LICENSE) altında sağlanmaktadır.
+
+## Katkıda Bulunanlar
+
+<a href="https://github.com/santifer/career-ops/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=santifer/career-ops" />
```

markdownlint-cli2 (0.23.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)